app/scripts/modules/core/instance/instance.write.service.js

```diff
@@ -133,16 +133,15 @@ module.exports = angular
       });
     }
 
-    function deregisterInstanceFromLoadBalancer(instance, application) {
+    function deregisterInstanceFromLoadBalancer(instance, application, params = {}) {
```

Comment:
@anotherchrisberry, I added the `params` parameter, which matches the functionality of other functions in this service.